basic/stam/stamExcersize/stam.py

```diff
@@ -0,0 +1,49 @@
+__author__ = 'Maya Gal-Yam'
```

Comment:
Delete this and disable this in your IDE
Usually code is edited by lots of people